src/components/Tags/_tags.scss

```diff
@@ -0,0 +1,113 @@
+@use '#scss/allPartials' as *;
+
+.#{$px}-tags-list {
+  display: flex;
+  flex-flow: row wrap;
+  gap: $spacing-sm;
+
+  .#{$px}-tag {
+    background-color: $pure-white;
+    border: 1px solid $button-hover;
+    border-radius: 6.25rem;
+    cursor: default;
+    gap: 0;
+    height: 2rem;
+    padding: 0 0.25rem 0 0.875rem;
+  }
+
+  .#{$px}-tag-label {
+    color: $button-hover;
+    font-size: 0.75rem;
+    font-variation-settings: 'wght' 600;
+    text-align: center;
+  }
+
+  .#{$px}-close-button {
+    padding-left: 0.2rem;
+
+    &:hover {
+      .#{$px}-tag-x {
+        background-color: $pure-white;
+        color: $pure-black;
+        font-variation-settings: 'wght' 600;
+      }
+    }
+
+    .#{$px}-tag-x {
+      color: $button-hover;
+      font-size: 0.75rem;
+      height: 1.75rem;
+      width: 1.75rem;
+
+      &:hover {
+        background-color: $pure-white;
+        color: $pure-black;
+      }
+
+      &:focus {
+        color: $pure-black;
+      }
+    }
+  }
+}
+
+.#{$px}-tags-list-clear-all-button {
+  border-color: $pure-white;
+  display: flex;
+  gap: 0;
+  height: 2rem;
+
+  &:hover {
+    cursor: pointer;
+
+    .#{$px}-left-arrow {
+      svg {
+        fill: $pure-black;
+
+        path {
+          fill: $pure-black;
+        }
+      }
+    }
+  }
+
+  &:focus-visible {
+    border-color: $button-hover;
+    border-radius: 6.25rem;
+    gap: 0;
+    outline-offset: 0;
+    padding: 0;
+  }
+
+  .#{$px}-left-arrow {
+    height: 1.25rem;
+    transform: scaleX(-1);
+    width: 1.25rem;
+
+    &:hover {
+      background: $pure-white;
+
+      svg {
+        fill: $pure-black;
+
+        path {
+          fill: $pure-black;
+        }
+      }
+    }
+
+    &:focus-visible {
+      &:focus-visible {
+        outline: none;
+        outline-offset: none;
+      }
+    }
+  }
+
+  .#{$px}-label {
+    color: $pure-black;
+    font-size: 0.875rem;
+    font-variation-settings: 'wght' 600;
+    padding-right: 0.5rem;
```

Comment:
I don't see this in the Figma

Reply:
This is to overwrite the CSS of children inside of the Button